### PR TITLE
fix(ui): probe label display on probes page

### PR DESCRIPTION
**Pull request by electron0zero**
Fix display of labels on probes list page.

Before:
image

After:
image

### CODE REVIEW

**Review by samjewell — APPROVED** [review]
Nice, thanks for this!
Seems to be failing tests.

File: src/components/ProbeList.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const ProbeList = ({ probes, onAddNew, onSelectProbe }: Props) => {
                 <span className="add-data-source-item-text">{probe.name}</span>
                 <span className="add-data-source-item-desc">
                   <Badge color={color} icon={onlineIcon} text={onlineTxt} />
-                  <div>{labelsToString(probe.labels)}</div>
+                  <div>{labelsToString(probe.labels).join(', ')}</div>
```
Comment (jewbetcha) [inline src/components/ProbeList.tsx:57]: [nit] this `.join()` could be chained onto the end of `labelsToString` if you want

Comment (electron0zero) [reply src/components/ProbeList.tsx]: updated

Comment (jewbetcha) [reply src/components/ProbeList.tsx]: 🔥

**Pull request merged**